Nodes/WaypointNode.cpp

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************************
+ *
+ * File:
+ * 		WaypointNode.cpp
+ *
+ * Purpose:
+ *		The WaypointNode sends information about the waypoints to the sailing logic
+ *
+ * Developer Notes:
+ *
+ *
+ ***************************************************************************************/
+
+#include "WaypointNode.h"
+#include "Messages/WaypointDataMsg.h"
+#include "logger/Logger.h"
+#include "dbhandler/DBHandler.h"
+#include <string>
+#include <vector>
+
+WaypointNode::WaypointNode(MessageBus& msgBus, DBHandler& db)
+: Node(NodeID::Waypoint, msgBus), m_db(db)
+{
+    m_id = 0;
+    m_longitude = 0;
+    m_latitude = 0;
+    m_declination = 0;
```

Comment:
I think we should come back to that later. Lets not pile on stuff until we are sure its working.